restic check with snapshot filters (continuation of PR #5213) (#5469)
restic/restic: doc/077_troubleshooting.rst

them using the ``repair pack`` command. Use that command instead of the "Repair the
index" section in this guide.

> If you are interested to check the repository via snapshots, you can now

Reviewer: ```suggestion
If you are interested to check only specific snapshots, you can now
```
